src/ert/gui/model/node.py

```diff
@@ -80,6 +80,7 @@ class RealNodeData:
     real_status_color: Optional[QColor] = None
     current_memory_usage: Optional[int] = None
     max_memory_usage: Optional[int] = None
+    stderr: Optional[str] = None
```

Comment:
What happens if 10gigabytes is dumped to the stderr file?

Reply:
:fire: Probably chaos...
I remember @xjules did a trick to only load a small part of the file if it is too large. I will see if I can find that again!

Reply:
I set the max size to 1500 as this should only be seen as a preview of the error file.
![image](https://github.com/user-attachments/assets/13179dce-3558-4fbc-9f49-9eb20b033e24)

Reply:
The lsf_driver has a function `read_stdout_and_stderr_files` which is related, but the driver might not be readily available to the gui.